app/buck2_common/src/init.rs

```diff
@@ -13,8 +13,8 @@ use std::time::Duration;
 use allocative::Allocative;
 use anyhow::Context;
 use buck2_core::buck2_env;
-use buck2_error::conversion::from_any_with_tag;
 use buck2_error::BuckErrorContext;
+use buck2_error::conversion::from_any_with_tag;
```

Comment:
@thoughtpolice  the internal lint format check is complaining about this import reordering, breaks format lint rules.   I can't fix it without taking over the imported diff and thus not being able to approve it, so best if you revert it and push again